server/database/queries/index.js

```diff
@@ -1,9 +1,11 @@
 const getAllProducts = require('./getAllProducts');
 const getClients = require('./getClients');
 const getOrders = require('./getOrders');
+const getFavorite = require('./getFavorite');
 
 module.exports = {
   getClients,
   getOrders,
   getAllProducts,
+  getFavorite,
 };
```

Comment:
please arrange all related routes by their category (common routes  first , then buyer and finally seller)
```suggestion
const getAllProducts = require('./getAllProducts');
const getFavorite = require('./getFavorite');
const getClients = require('./getClients');
const getOrders = require('./getOrders');

module.exports = {
  getAllProducts,
  getFavorite,
  getClients,
  getOrders,
};
```